### PR TITLE
Implement description updating

### PR DESCRIPTION
**Pull request by yet-another-account**
Repost of #222

### CODE REVIEW

**Comment by yet-another-account** [comment]
Fixed merge conflict

File: github_api/repos.py [hunk 1]
```diff
@@ -7,3 +7,12 @@ def get_num_watchers(api, urn):
     # which always matches "stargazers_count"
     return data["subscribers_count"]
 
+
+def set_desc(api, urn, desc):
+    """ Set description of repo """
+    path = "/repos/{urn}".format(urn=urn)
+    data = {
+        "name": "chaos",
```
Comment (reddraggone9) [inline github_api/repos.py]: I feel like the name should probably come from settings, but oh well.

**Review by reddraggone9 — APPROVED** [review]
[Relevant API endpoint](https://developer.github.com/v3/repos/#edit) and 👍

**Comment by phil-r** [comment]
Is it possibe to modify repo url? it would make sense to set it to chaosthebot.com

**Comment by reddraggone9** [comment]
But that's not the repo URL. That's where the code is running...

**Comment by phil-r** [comment]
I mean `homepage` in terms of github and here is the link, that anwers my question: https://developer.github.com/v3/repos/#edit

**Comment by yet-another-account** [comment]
There, used repo name from settings.

File: settings.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 # "chaosbot/chaos"
 URN = misc.get_self_urn()
 GITHUB_USER = URN.split("/")[0]
+GITHUB_REPO = URN.split("/")[1]
```
Comment (reddraggone9) [inline settings.py]: **MOAR pythonic**!
```python
GITHUB_USER, GITHUB_REPO = URN.split("/")
```

**Comment by phil-r** [comment]
@reddraggone9 I'm so blind, you posted link above 🤦‍♂️

**Comment by yet-another-account** [comment]
Okay, I put them on the same line.

**Pull request merged**